tests/fixtures/test_recommendations_for_authors/content_01_expected.py

```diff
@@ -0,0 +1,3 @@
+from collections import OrderedDict
+
+expected = OrderedDict()
```

Comment:
just a curious question, no need to change anything, but in these cases with empty expectations what is `content_01` vs `content_02`? is it the old kitchen sink vs the new one? 

and in this specific case, if this fixture is being used for `test_recommendations_for_authors` why are we testing against an `editorReport`? 

From this:
https://github.com/elifesciences/XML-mapping/blob/a559aaa70ac6239528e9ecabe0d2e69802337a95/elife-1234567890-v2/elife-1234567890-v2.xml#L2005

it does look like an "editor's report" changed to an "editor's evaluation" that then changed to an "elife assessment" ... actually I just took a look at the other fixture and it looks like a copy+paste job so just ignore me ;)

Reply:
I wasn't too picky about the XML fixtures in some of these test cases. The old kitchen sink `content_01` test here is an example of an XML which has a `<sub-article>` tag and will not return `Recommendations for authors` content.

It is probably a bit of overkill in this case, but it is slightly different than parsing `<root/>` (which doesn't contain any `<sub-article>` tags), and the `content_02` test is an example which will return a `Recommendations for authors`. I threw in some test cases on the old kitchen sink style XML to test for empty return values and guard against regressions.